tests/org.jboss.tools.central.ui.bot.test/pom.xml

```diff
@@ -14,13 +14,7 @@
 	<packaging>eclipse-test-plugin</packaging>
 
 	<properties>
-		<jbosstools.test.wildfly.home>${requirementsDirectory}/wildfly-10.0.0.Final</jbosstools.test.wildfly.home>
-		<wildflySystemProperties></wildflySystemProperties>
-		<eapSystemProperties></eapSystemProperties>
-		<jbosstools.test.jboss-eap-6.4.home>${requirementsDirectory}/jboss-eap-6.4</jbosstools.test.jboss-eap-6.4.home>
-		<jbosstools.test.jboss-eap-7.0.home>${requirementsDirectory}/jboss-eap-7.0</jbosstools.test.jboss-eap-7.0.home>
-		<rd.config>${project.build.directory}/classes/server_properties</rd.config>
-		<systemProperties>${integrationTestsSystemProperties} ${wildflySystemProperties} ${eapSystemProperties} -Dusage_reporting_enabled=false -Drd.config=${rd.config} -Dmaven.config.file=${project.build.directory}/classes/settings.xml</systemProperties>
+		<systemProperties> -Dusage_reporting_enabled=false -Dmaven.config.file=${project.build.directory}/classes/settings.xml</systemProperties>
```

Comment:
-Dmaven.config.file.. where is it used ? I see that you are using target/classes/settings.xml in your tests but not this property. So either you should use the property and change the test or you dont need this prop at all